app/views/includes/contributions_earnings.scala.html

```diff
@@ -88,4 +65,4 @@ <h3 class="govuk-heading-s"  id="gmp-conts-and-earnings-header-@count">
         }
     }
 
-<hr class="govuk-section-break govuk-section-break--m" aria-hidden = true/>
+<br>
```

Comment:
Avoid using br tags - if additional spacing is need then there are override classes which can do this.